heufybot/modules/commands/pronouns.py

```diff
@@ -28,7 +28,7 @@ def lookUpPronouns(self, server, source, user, displayErrors):
                 "pronouns": self.pronstore[networkName(self.bot, server)][user.lower()]
             }
         if displayErrors:
-            error = "User's pronouns have not been specified. Would you like to add your own, {}?".format(user)
+            error = "User's pronouns have not been specified. Your pronouns can be set with ~setpron, {}".format(user)
```

Comment:
The command character should not be hardcoded as it is configurable. This should probably be formatted like I did it in https://github.com/Heufneutje/PyHeufyBot/blob/master/heufybot/modules/commands/userlocation.py#L31 for consistency sake, leaving the command character out altogether.